src/InventoryManager.cpp

```diff
@@ -39,17 +40,31 @@ int InventoryManager::userInput() {
             std::cin.clear();
             std::cin.ignore(10000, '\n');
 
-            std::cout << "Name Perishable/NonPerishable SubCategory Quantity ID SalePrice "
-                         "CostOfGood Tax ExpirationDate (Enter -1000000 if unsure): ";
-            std::cin >> name >> category >> sub_category >> quantity >> id >> sale_price >> buy_price >> tax >>
-                expiration;
+            std::cout << "Enter item name: ";
+            std::cin >> name;
+            std::cout << "Enter item category (Perishable or NonPerishable): ";
+            std::cin >> category;
+            std::cout << "Enter item sub-category: ";
+            std::cin >> sub_category;
+            std::cout << "Enter item quantity: ";
+            std::cin >> quantity;
+            std::cout << "Enter item id: (up to 6 digits) ";
+            std::cin >> id;
+            std::cout << "Enter sale price: (format xx.xx) $";
+            std::cin >> sale_price;
+            std::cout << "Enter purchase cost: (format xx.xx) $";
+            std::cin >> buy_price;
+            std::cout << "Enter item tax: (format xx.xx) $";
```

Comment:
Tax is entered as a decimal and calculated by the program.

So: 0.1 == 10% tax.

If you want you could change this to something like:
```
std::cout << "Enter item tax: (% of 100%) ";
```
and then just do:
```
tax /= 100;
```

Reply:
Also, $ come before the values with money, unless you don't want them to put the $ in which case it would be best not to include it at all there.

Reply:
I'm going to say enter tax as a decimal value. does that work? and brenaren, the dollar sign is so that it looks nicely formatted when the user types in the numbers(the $ is already there so if user types in 2.22, it displays as $2.22.  i think it looks nicer this way but i can take it out if need be